monocle/github/pullrequest.py

```diff
@@ -336,6 +337,7 @@ def extract_pr_objects(pr):
             change['text'] = pr['bodyText']
             change['additions'] = pr['additions']
             change['deletions'] = pr['deletions']
+            change['approval'] = pr['reviewDecision']
```

Comment:
we should make it a list like in gerrit  to make the code easier at the API level.

Reply:
Done